app/models/concerns/itemizable.rb

```diff
@@ -102,14 +102,21 @@ def total_quantity
     line_items.total
   end
 
-  def to_a
+  def line_item_values
     line_items.map do |l|
-      # When the item isn't found, it's probably just inactive. This ensures it's available.
       item = Item.find(l.item_id)
       { item_id: item.id, name: item.name, quantity: l.quantity, active: item.active }.with_indifferent_access
     end
   end
 
+  def to_a
+    line_item_hashes unless Flipper.enabled?(:deprecate_to_a)
```

Comment:
This method was renamed, wasn't it? :smiley:

Also, there's no `return` keyword here, so it will always raise an error regardless of the Flipper value.

Reply:
Could we add tests for this method so we can validate what happens when the flag is both on and off?